src/lib.rs

```diff
@@ -74,21 +74,28 @@ extern crate error_chain;
 #[macro_use]
 extern crate serde_derive;
 extern crate serde;
-#[macro_use] 
+#[macro_use]
 extern crate serde_json;
+#[macro_use]
+extern crate log;
 
 extern crate handlebars;
 extern crate pulldown_cmark;
 extern crate regex;
 
+#[cfg(test)]
 #[macro_use]
-extern crate log;
+extern crate pretty_assertions;
+#[cfg(test)]
+extern crate tempdir;
```

Comment:
`tempdir` is very handy in tests so I guess that we could make the import it localized to them as long as it's used only in one suite. If you plan to use it elsewere (or in other suites) then its fine where it is.